--- rtsp_3.7-v2.original/nf_conntrack_rtsp.c	2014-01-26 21:57:26.000000000 +0100
+++ rtsp_3.7-v2.nuevo/nf_conntrack_rtsp.c	2014-10-19 12:56:31.850066591 +0200
@@ -346,7 +346,7 @@

 		nf_ct_expect_init(rtp_exp, NF_CT_EXPECT_CLASS_DEFAULT,
 				  nf_ct_l3num(ct),
-				  &ct->tuplehash[!dir].tuple.src.u3,
+				  NULL, //&ct->tuplehash[!dir].tuple.src.u3,
 				  &ct->tuplehash[!dir].tuple.dst.u3,
 				  IPPROTO_UDP, NULL, &be_loport);

@@ -364,7 +364,7 @@

 			nf_ct_expect_init(rtcp_exp, NF_CT_EXPECT_CLASS_DEFAULT,
 					  nf_ct_l3num(ct),
-					  &ct->tuplehash[!dir].tuple.src.u3,
+					  NULL, //&ct->tuplehash[!dir].tuple.src.u3,
 					  &ct->tuplehash[!dir].tuple.dst.u3,
 					  IPPROTO_UDP, NULL, &be_hiport);
            
